Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -64,33 +64,6 @@ button:hover, input[type=submit]:hover, input[type=button]:hover, input[type=res
     background-color: -libweb-palette-hover-highlight;
 }
 
-input[type=checkbox] {
```

Comment:
Commit message: I don't know what "animalised" means but the definitions I've found don't make sense here so I'm assuming it's a typo?